### PR TITLE
feat: Add booking contact data for flexible transport

### PR DESCRIPTION
**Pull request by jorelosorio**
Related to https://github.com/AtB-AS/kundevendt/issues/3102

### CODE REVIEW

**Review by rosvik — APPROVED** [review]

File: src/service/impl/trips/converters.ts [hunk 1]
```diff
@@ -68,6 +68,7 @@ function mapToLegacyLeg(leg: Leg_v2): Leg {
       leg.serviceJourney
     ) as ServiceJourney,
     situations: [],
+    bookingArrangements: undefined,
```
Comment (jorelosorio) [inline src/service/impl/trips/converters.ts:71]: @rosvik I had to add this because of an issue, do you know if that is OK? Or what that legacy is about.

**Comment by jorelosorio** [comment]
@rosvik I had to merge recent changes on code from your side, could you please double-check those are included? Thanks :)

File: src/service/impl/trips/converters.ts (hunk 1 above)
Comment (rosvik) [reply src/service/impl/trips/converters.ts:71]: That looks fine! This is to support clients that haven't updated to an app version with OTP2 travel search. It should be removed at some point, but for now, it's just to make sure old clients get somewhat useful data, and don't crash.

**Review by rosvik — APPROVED** [review]
Still LGTM 👍

**Comment by tormoseng** [comment]
Will be tested as part of the Flexible transport initiative.

**Comment by tormoseng** [comment]
This is now in Prod

**Pull request merged**